content/gamenews

```diff
@@ -1,4 +1,4 @@
-2015-08-30 03-00-00:
-	Title: Playtest 20150830
-	Author: Pavel Penev
-	Content: We are happy to announce that another playtest on the way to our next release is now available!\n\nThis one focused on fixing bugs and polishing the existing features.\n\nHave a look at http://openra.net for details!\n\nAs always with our playtests, we invite you to report any issues you encounter at http://bugs.openra.net to ensure the final release is as bug-free as possible. Thanks, and enjoy!
+2015-09-12 12-00-00:
+	Title: Release 20150919
+	Author: Paul Chote
+	Content: LANDCOM 16 HQS.\nTOP SECRET\nTO: ALL FIELD COMMANDERS\n\nAN INTERCEPTED COMMUNIQUE INDICATES THE\nIMMEDIATE AVAILABILITY OF A NEW SEPTEMBER\n2015 OPENRA RELEASE.  SUBSEQUENT INTERROGATION OF THE MESSAGE COURIER REVEALED THAT 31 AUTHORS CONTRIBUTED\nMORE THAN 1150 CHANGES TOWARDS THEIR\nWAR EFFORT IN THE LAST THREE MONTHS.  A LARGE COLLECTION OF BUG FIXES AND OTHER\nIMPROVEMENTS ARE SUSPECTED.\n\nYOUR ORDERS ARE TO PROCEED DIRECTLY\nTO THE OPENRA WEBSITE AND DOWNLOAD THE NEW RELEASE.  RENDEZVOUS WITH OTHER COMMANDERS IN-GAME AND DESTROY ALL ENEMY OPPOSITION.\n\nTRANSMISSION ENDS.
```

Comment:
*ugh*.

I set this up using the TD dropdown, thinking this would be consistent across mods (is this too much to ask?).

![screen shot 2015-09-19 at 12 33 50](https://cloud.githubusercontent.com/assets/167819/9976011/e0dd39d4-5eca-11e5-9c3a-20e1a4af949b.png)